dora/core/server/master/src/main/java/alluxio/master/job/CopyJobFactory.java

```diff
@@ -33,33 +33,31 @@
  */
 public class CopyJobFactory implements JobFactory {
 
-  private final UnderFileSystem mFs;
+  private final DefaultFileSystemMaster mFs;
```

Comment:
Why use `DefaultFileSystemMaster` instead of `UnderFileSystem`?

Reply:
cause I need master Ufs Manager